crates/ide_completion/src/config.rs

```diff
@@ -15,4 +15,5 @@ pub struct CompletionConfig {
     pub add_call_argument_snippets: bool,
     pub snippet_cap: Option<SnippetCap>,
     pub insert_use: InsertUseConfig,
+    pub label_details: bool,
```

Comment:
It feels like this is strictly LSP-specific, and affects only how we convert `ide::CompletionItem` to `lsp_types::CompletionItem`. So, this setting should only exist in the `rust-analyzer` crate.

Reply:
I'm not so sure. For instance: if the client supports label details and we have on the fly imports enabled it may make sense to move the `(use import::path)` out of the label and render it as part of the `CompletionItemLabelDetails`